modules/public/views.py

```diff
@@ -179,14 +179,22 @@ def snap_details(snap_name):
     country_data = []
     try:
         today = datetime.datetime.utcnow().date()
-        week_ago = today - relativedelta.relativedelta(weeks=1)
+        yesterday = today - relativedelta.relativedelta(days=1)
+        country_metric_name = 'installed_base_by_country_percent'
+        os_metric_name = 'weekly_installed_base_by_operating_system_normalized'
 
         metrics_query_json = [
             {
-                "metric_name": "installed_base_by_country_percent",
+                "metric_name": country_metric_name,
                 "snap_id": details['snap_id'],
-                "start": week_ago.strftime('%Y-%m-%d'),
-                "end": today.strftime('%Y-%m-%d')
+                "start": yesterday.strftime('%Y-%m-%d'),
```

Comment:
Why was query start/end changed from week/today to yesterday/yesterday?
Wouldn't it affect the data displayed on map and chart? Is it desired?

Reply:
While talking to Celso he said: today doesn't have any data
I also did a test, and the shading didn't change when I switched to yesterday/yesterday and it makes the request/response from the api a little bit smaller and quicker!